ignition/service/lifecycle.py

```diff
@@ -55,6 +77,11 @@ def get_lifecycle_execution(self, request_id, deployment_location):
         :param str request_id: identifier of the request to check
         :param dict deployment_location: the deployment location the Resource is assigned to
         :return: an ignition.model.lifecycle.LifecycleExecution
+        
+        :raises:
+            ignition.service.lifecycle.LifecycleExecutionRequestNotFoundError: if no request with the given request_id exists
+            ignition.service.lifecycle.TemporaryLifecycleError: there is an issue handling this request at this time, an attempt should be made again at a later time
```

Comment:
"an attempt should be made again at a later time" implies that it's the responsibility of the client to do this, maybe say that Ignition will re-attempt the request at a later time?

Reply:
Because this is the interface, it's supposed to be that the caller should try again later. I can add a comment about Ignition on there though.